src/shared-components/dropdown/mobileDropdown.tsx

```diff
@@ -7,21 +7,27 @@ import { DropdownContainer, dropdownInputStyle, IconContainer } from './style';
 import { OptionType } from './index';
 
 type MobileDropdownProps = {
-  value: string | number | null;
-  options: OptionType[];
-  textAlign: 'left' | 'center';
+  borderRadius: string;
   onSelectChange: (event: React.ChangeEvent<HTMLSelectElement>) => void;
+  options: OptionType[];
+  textAlign?: 'left' | 'center';
+  value?: string | number | undefined;
 };
 
 const MobileDropdown = ({
```

Comment:
CHANGELOG:

1. Change default `value` and `options[number].value` type from `null` to `undefined` to conform to HTML attribute types. 
1. Remove defaultProps for `onSelectChange`
    - I'm assuming this was included to make the tests easier--it's a required TypeScript type so this makes things more consistent. 
1. Sets `textAlign` TypeScript type as optional to reflect that we have an ES6 default. 
    - This isn't consistent with parent `Dropdown` behavior but I didn't want to remove this functionality. All sub-component usage goes through `Dropdown`.